python/defs.bzl

```diff
@@ -21,26 +21,22 @@ migrated to Starlark, their implementations will be moved here.
 
```

Comment:
the comment above feels like it should go to the new file, and the comment here should just be "public API" or sth